chain/actors/builtin/miner/v2.go

```diff
@@ -83,9 +83,12 @@ func (s *state2) PreCommitDeposits() (abi.TokenAmount, error) {
 
 func (s *state2) GetSector(num abi.SectorNumber) (*SectorOnChainInfo, error) {
 	info, ok, err := s.State.GetSector(s.store, num)
-	if !ok || err != nil {
+	if err != nil {
 		return nil, err
 	}
+	if !ok {
+		return nil, xerrors.Errorf("sector %d not found", num)
+	}
```

Comment:
This isn't correct, some callers of this method expect a nil value when the sectors isn't found

Reply:
Got it. I modified the code elsewhere instead, please review it again.